adresse/controllers/service.classic.php

```diff
@@ -260,32 +260,54 @@ function export(){
 
     $fileName = '';
     $name = '';
+    $type = '';
 
     if($result != Null){
       if ($option == 'bal'){
+        $type = 'binary';
         $fileName = tempnam($tempPath, 'exportbal-');
```

Comment:
```suggestion
        $fileName = tempnam($tempPath, 'exportbal-'.$insee.'-'.time());
```
C'est une suggestion, la methdoe time() peut-être remplacer par une autre variable